src/main/java/binaries/maday/BinaryArray.java

```diff
@@ -18,6 +18,14 @@ public BinaryArray(int[] binaries) {
      * Returns the decimal numeric value equivalent to the list of binaries stored
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        //throw new UnsupportedOperationException("Not supported yet.");
+        int potencia = binaries.length-1;
+		int decimal = 0;		
+		for (int i : binaries) {
+			int convertidos= i*(int)(Math.pow(2, potencia));
```

Comment:
let's change `convertidos` to english. Missing spaces before and after `*` operator. `2` looks like magic number.